src/app/core/tools/handlers/PasteHandler.ts

```diff
@@ -7,9 +7,12 @@ export const PasteHandler = (paste: (text: string) => boolean): EventHandler =>
     return ({
         conditions: (event: Event) =>
             (event.type === "paste" &&
+             (event.ev.clipboardData != undefined) &&
```

Comment:
is this necessary because it complains about `event.ev.clipboardData.getData` below?

can you not just do `event.ev.clipboardData?.getData`?

Reply:
You cannot